mempool/reactor.go

```diff
@@ -19,7 +17,7 @@ import (
 const (
 	MempoolChannel = byte(0x30)
 
-	aminoOverheadForTxMessage = 8
+	protoOverheadForTxMessage = 4
```

Comment:
Interesting, how is this even shorter? Ah, nvm, I see, it's because the prefix bytes that are gone now. Forgot that this was an interface.

Reply:
I was thinking about this as well but my reason for not doing this is that if we were to add a new type then we would like to add logic to accompany it, this would result in more code needing to be written so it would not be hard to do it then, but also we do have a refactor planned for the mempool so I cant see new functionality being added to this version.